.gitignore

```diff
@@ -1,6 +1,3 @@
-# use glob syntax.
-syntax: glob
-
```

Comment:
As I understand it, this is totally meaningless in a .gitignore file (I suspect it got copied from an .hgignore file)